db/migrate/20250309121956_rename_ansible_tower_fqdn_to_api_url.rb

```diff
@@ -0,0 +1,19 @@
+class RenameAnsibleTowerFqdnToApiURL < ActiveRecord::Migration[7.0]
+  def up
+    Parameter.where(name: 'ansible_tower_fqdn').find_each do |parameter|
+      parameter.update(
+        name: 'ansible_tower_api_url',
+        value: "https://#{parameter.value}/api/controller/v2"
+      )
+    end
+  end
+
+  def down
+    Parameter.where(name: 'ansible_tower_api_url').find_each do |parameter|
+      parameter.update(
+        name: 'ansible_tower_fqdn',
+        value: parameter.value.sub('https://', '').sub('/api/controller/v2', '')
```

Comment:
This approach wouldn't work for variables whose values have been modified. Should we also account for cases where the API path of the variable has been updated?

Reply:
I'd use [URI](https://rubyapi.org/o/uri) to extract the host of the URL.
```suggestion
        value: URI(parameter.value).host
```